dev-php/jsonlint/jsonlint-1.4.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="JSON Lint for PHP"
+HOMEPAGE="https://github.com/Seldaek/jsonlint"
+SRC_URI="https://github.com/Seldaek/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="dev-lang/php:*
+	=dev-php/fedora-autoloader-0.2.1"
+DEPEND="test? ( ${RDEPEND}
+				dev-php/phpunit )"
```

Comment:
Entirely your choice. The only thing I'm complaining about is using more than one tab for indent (nesting) level. The idea is to do things like:

```
FOO="
<1-tab>    bar                 # variable scope
<1-tab>    baz? (
<2-tab>        bazbar          # 1 nested block scope
<2-tab>        bazbaz? (
<3-tab>             foobaz     # 2 nested blocks scope
...
```

It's fine to close braces either on the same line as last text, or in line following it with indent matching opening brace keyword. However, the latter is recommended when the conditions become complex.